docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -100,128 +117,212 @@ In the case of errors, include human-readable error messages in the response pay
+- Configuration fields: 
 
-## API installation workflow
+  If you would like [configuration fields](#configuration-fields) to be set up for your carrier, specify the properties you would like to use for these connection and/or settings options. 
 
-To set up a carrier using the API, first, connect it using the Connect Carrier API. Make a request containing the connection settings required by your carrier. The ID of the carrier is required. The carrier ID will be issued by BigCommerce when your carrier is registered. All connection fields are unique to each carrier. If your carrier doesn’t require any connection settings, send an empty object for the `connection` settings property.
+  We will then send you a `code` for each configuration option. API users will specify the `code` for connections options when they [connect your carrier to their store](#how-your-app-will-be-connected-to-a-store). BigCommerce will also include the `code` for each connection option when we [request rates from your carrier](#provide-shipping-rates-to-bigcommerce). API users specify the `code` for settings options when they define your carrier's shipping method for their shipping zones.  
 
-```http title="Example carrier connection request with connection settings" lineNumbers
-POST https://example.com/shipping/carrier/connection
+## What's next?
+
+### How your app will be connected to a store
+
+Once a merchant installs your app on their store, merchants and API users can connect your carrier to their store by using the connection options that you provided when you submitted your app.    
+
+#### How merchants will use your app
+A merchant can navigate to the Shipping Manager UI to input carrier connection settings. The UI displays the connection option's `label` that you provided when you submitted your app. The following figure shows how connection options are shown to merchants in the control panel, if you provided a text field for API Key and a checkbox field for enabling expedited shipping:
+
+![Connect Carrier via UI](https://storage.googleapis.com/bigcommerce-production-dev-center/images/connection%20settings.png 'Connection options in Shipping Manager') 
+
+After connecting your carrier, a merchant can define and enable multiple real-time shipping methods for your carrier app in any shipping zone. 
+
+#### How API users will use your app
+API users can connect your carrier to the store by using the [Create a carrier connection](/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint. In the request, API users will send the carrier ID that you obtained during app setup, as well as values for your app's connection options. Specifically, API users will specify the `code` for each connection option as a property under the `connection` object when they connect to your carrier, for example:    

```

Comment:
Should we recommend that shipping provider devs document their connection option `codes` for third parties that may want to programmatically set up a connection?

Reply:
I have added a note/sentence in 2 places to recommend this.